### ISSUE TITLE
View subgenerator not working

### ISSUE THREAD

**Issue by ghost** [srsgores/generator-joomla-component#1]
When I try to generate a new view by doing

```
yo joomla-component:view test
```

I get

```
You called the view subgenerator with the argument test.
create somefile.js
```

I.e. the only thing that happens is that a file called 'somefile.js' is created in the component's root directory. Am I doing something wrong?

**Comment by g13013** [comment]
Same here, view, helper and controller generate somefile.js

**Comment by srsgores** [comment]
@sir-sure-e-nuff and @g13013 , I regret to inform you that subgenerators are not yet complete in this repository.  If you [take a look at the source files in the subgenerators](https://github.com/srsgores/generator-joomla-component/blob/master/controller/index.js), you may notice that they are simply placeholders and not yet complete.

I invite any of you to make a pull request.  I am currently too busy with other work to finish the subgenerators at this moment, but I will have a look when time permits.

Thanks for posting.

**Comment by srsgores** [comment]
Fixed in ff3efcb1fa76861471e3495ce102e72bae662b82
